### PR TITLE
Implement empty ticks list endpoint.

### PR DESCRIPTION
**Pull request by LINCKODE**

### CODE REVIEW

File: validator/tick/empty_tick.go [hunk 1]
```diff
@@ -14,19 +14,19 @@ import (
 
 var emptyTickData = &protobuff.TickData{}
 
-func CalculateEmptyTicksForEpoch(ctx context.Context, ps *store.PebbleStore, epoch uint32) (uint32, error) {
+func CalculateEmptyTicksForEpoch(ctx context.Context, ps *store.PebbleStore, epoch uint32) ([]uint32, error) {
```
Comment (0xluk) [inline validator/tick/empty_tick.go:17]: can you add some tests for this empty ticks logic? something like calling it once and multiple times and check the db to see that the results are the expected ones
